### DISCUSSION TITLE
Is Android 4.4 getting too old for KOReader support?

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#7916] [Q&A] [answered]
As the title goes, is it getting too old in foreseeable future for KOReader support?
I'm eyeing a few options of e-ink readers with Android, some of which have only 4.4 .
Thanks!

**Comment by Frenzie** [comment]
Yes, I'd definitely try to get something with at least Android 7 if at all possible. Android 7 is five years old by now. Google is currently popping up warnings for Android 5.1 (API level 22) or lower, so anything older than Android 6 is unfortunately already living on borrowed time.

Pinging @pazos for a more informed view.

**Reply by mergen3107** [reply]
Thank you!

**Comment by pazos** [comment, marked as answer]
> Is Android 4.4 getting too old for KOReader support?

Yes. At some point anything below Marshmallow/Nougat will be deprecated. That would mean no new builds, but current builds will work fine (or as fine as they work now, which seem to be device dependant).

> I'm eyeing a few options of e-ink readers with Android, some of which have only 4.4

Never played with Tolinos. They seem to work fine. They're very well supported by @zwim.
Nooks should work almost the same.

My suggestion would be a Likebook if you don't care a lot about battery life on suspension (Rockchip limitation?)

If you do care about battery get a Onyx Boox with a Qualcomm processor. Some onyx devices are broken from the PoV of KOReader,  so choose one that's currently supported (like @Galunid's nova2)

**Reply by mergen3107** [reply]
Thank you so much! Great details

**Comment by mergen3107** [comment]
Guys, do you know if Onyx Boox Nova (the first one) is well supported by KOReader? It has ARM Cortex-A17 1.6GHz Quad-core processor, Android 6 and 2 GB or RAM.

I couldn’t find issues, open and closed, related to that specific model. 

Thanks!

**Reply by Frenzie** [reply]
The Onyx EPD controller (E Ink) was written against a Nova 2 afaik. It would be logical if it were very similar but I don't know.

**Reply by mergen3107** [reply]
Thank you!

**Reply by Galunid** [reply]
@mergen3107 It should be, you should be able to enable EPD controller by adding the device in https://github.com/koreader/android-luajit-launcher/blob/master/app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt and https://github.com/koreader/android-luajit-launcher/blob/master/app/src/main/java/org/koreader/launcher/device/EPDFactory.kt if you wish, but it shouldn't have any trouble working without it.

**Reply by mergen3107** [reply]
Thanks!
